File: fizzbuzz.m
//
//  fizzbuzz.m

#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {

    for (int i = 1; i < 101; i++) {
        if (i % 15 == 0) {
            NSLog(@"fizzbuzz");
        } else if (i % 5 == 0) {
            NSLog(@"buzz");
        } else if (i % 3 == 0) {
            NSLog(@"fizz");
        } else {
            NSLog(@"%i", i);
        }
    }
    
    return 0;
}

File: fizzbuzz.php
<?php
for ($x = 1; $x < 101; $x++) {
  if ($x % 15 == 0) {
    echo "fizzbuzz\n";
  } else if ($x % 5 == 0) {
    echo "buzz\n";
  } else if ($x % 3 == 0) {
    echo "fizz\n";
  } else {
    echo "$x\n";
  }
}
